lib/googleanalytics.jsx

```diff
@@ -0,0 +1,27 @@
+var React = require('react');
+var config = require('./config');
+
+var GoogleAnalyticsBasicTag = React.createClass({
+  render: function() {
+    if (config.GA_ACCOUNT) {
+      return (
+        <script dangerouslySetInnerHTML={{
+          __html: "(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ \
```

Comment:
Ahh, that's interesting. 

But do you know if this would this reload `www.google-analytics.com/analytics.js` on every page load?